classes/api/usage.php

```diff
@@ -0,0 +1,51 @@
+<?php
+// This file is part of the QuestionPy Moodle plugin - https://questionpy.org
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace qtype_questionpy\api;
+
+use qtype_questionpy\array_converter\array_converter;
+use qtype_questionpy\array_converter\converter_config;
+
+defined('MOODLE_INTERNAL') || die;
+
+/**
+ * Server usage.
+ *
+ * @package    qtype_questionpy
+ * @author     Alexander Schmitz
+ * @copyright  2023 TU Berlin, innoCampus {@link https://www.questionpy.org}
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class usage {
+
+    /** @var int */
+    public int $requestsinprocess;
+
+    /** @var int */
+    public int $requestsinqueue;
```

Comment:
Analog hier: Uninitialisierte Properties sind nicht so geil, lieber in den Konstruktor aufnehmen, damit sie required sind, oder default `null` setzen und die Zeile in der Template in dem Fall nicht anzeigen.

Reply:
Ich hab mich jetzt dafür entschieden beide in den Konstuktor aufzunehmen, da das auch dem `Usage` model im questionpy-server entspricht.